aws.yaml

```diff
@@ -146,6 +146,44 @@
     - name: cluster-operator
       provider: aws
       version: 2.0.0
+- version: 9.2.0
+  active: true
+  date: 2020-02-24T12:00:00Z
+  apps:
+    - app: cert-exporter
+      version: 1.2.1
+    - app: chart-operator
+      version: 0.11.4
+    - app: cluster-autoscaler
+      componentVersion: 1.16.2
+      version: 1.1.3
+    - app: coredns
+      componentVersion: 1.6.5
+      version: 1.1.3
+    - app: kube-state-metrics
+      componentVersion: 1.9.2
+      version: 1.0.2
+    - app: metrics-server
+      componentVersion: 0.3.3
+      version: 1.0.0
+    - app: net-exporter
+      version: 1.6.0
+    - app: nginx-ingress-controller
+      componentVersion: 0.29.0
+      version: 1.5.0
+    - app: node-exporter
+      componentVersion: 0.18.1
+      version: 1.2.0
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: aws-operator
+      version: 5.6.0
```

Comment:
Why chart operator is not listed in this last one as authority?

Reply:
@rossf7 explained in https://github.com/giantswarm/releases/pull/84#discussion_r381200033

If I'm not mistaken, same rationale applies for legacy aws releases like this too.